posts/app.py

```diff
@@ -36,11 +36,11 @@
 def upload_new_post():
     """Make a new post upload to the server.
 
-    Post request body should contain:
+    Post request body should contain multipart/form-data with:
     event_id: id of the event to post to
     author_id: user id of the user making the post
     text: text to be sent
-    Post request files should contain all the files the user wants to upload
+    All the files the user wants to upload
```

Comment:
Should these be in a list?

Reply:
No, you actually are supposed to upload files in a request by sending a multipart/form-data content type (-F in curl) and sending each parameter individually. I.e. to send multiple files in curl, `curl -F foo=bar -F "file_1=/path/to/file_1" -F "file_2=/path/to/file_2" -X https://url.com`. It doesn't matter the names for the keys of the files you give, as files are automatically recognized and passed to Flask as `request.files`.

Reply:
I've updated the design doc to reflect this.